.github/workflows/quicktest.yml

```diff
@@ -0,0 +1,88 @@
+name: Quicktest
+
+on:
+  # Run on pushes, including merges, to all branches except `master`.
+  push:
+    branches-ignore:
+      - master
+    paths-ignore:
+      - '**.md'
+  # Allow manually triggering the workflow.
+  workflow_dispatch:
+
+jobs:
+  #### QUICK TEST STAGE ####
+  # This is a much quicker test which only runs the unit tests and linting against low/high
+  # supported PHP/PHPCS/WPCS combinations.
+  quicktest:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        include:
+          - php: 'latest'
+            phpcs_version: 'dev-master'
+            wpcs_version: 'dev-master'
+          - php: 'latest'
+            phpcs_version: '3.6.0'
+            wpcs_version: '2.3.0'
+          - php: '5.4'
+            phpcs_version: 'dev-master'
+            wpcs_version: '2.3.0'
+          - php: '5.4'
+            phpcs_version: '3.6.0'
+            wpcs_version: 'dev-master'
+
+    name: "QTest${{ matrix.phpcs_version == 'dev-master' && ' + Lint' || '' }}: PHP ${{ matrix.php }} - PHPCS ${{ matrix.phpcs_version }} - WPCS ${{ matrix.wpcs_version }}"
```

Comment:
The `.php` got me confused for a moment, because it looks like a filename.

Is it possible to use `php_version` like `phpcs` and `wpcs` or is this a "magical" name that is being used somewhere else?

Reply:
Comment below regarding linting also applies on the condition here to add " + Lint" to the name of the action.

Reply:
> Is it possible to use php_version like phpcs and wpcs or is this a "magical" name that is being used somewhere else?

Yes, that's possible. The names used in the matrix are up to the script writer to determine.